### PR TITLE
add new acerep

### PR DESCRIPTION
**Pull request by bux**

### CODE REVIEW

File: docs/_posts/2017-07-11-ace3-version3100.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+With Jets DLC focusing primarily on ... Jets ... this update we also focused a little bit more on aircraft. As an example we added rearm features to the new pylon system introduced with Jets DLC, we added settings to increase the fuel hose's length (for refueling on the carrier) and removed our custom flight model changes in favour of the constantly improved ones from vanilla Arma 3.
+
+<div class="row">
+    <div class="small-12 columns">
+        <img src="{{site.baseUrl}}/img/news/170711_jet_rearm.jpg" alt="A pilot and two deck crew members rearming an F-18 Black Wasp."/>
+    </div>
+</div>
+
+Secondly Zeus has recieved new toys and improvements. One of which is to use AI to suppress positions and direct it using a slick new UI component we have developed for the Zeus interface.
```
Comment (Neviothr) [inline docs/_posts/2017-07-11-ace3-version3100.md]: It's `received`.

File: docs/_posts/2017-07-11-ace3-version3100.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Back in [ACEREP #00004]({{site.baseUrl}}/2016/03/02/ace3-version350.html) we last spoke about Intercept a project that hooks into the Arma 3 game engine and calls scripting commands on an engine level (in C++) thus circumventing the SQF interpreter and saving valuable time executing the commands.
+
+<div class="row">
+    <div class="small-4 columns">
+        <img src="{{site.baseUrl}}/img/news/170711_logo-intercept.png" alt="The intercept project's logo."/>
+    </div>
+</div>
+
+Though in june 2016 progress on Intercept had become stagnant as the main developer got started on his new job and time for intercept had become rather sparse. In may 2017 [@dedmen](https://github.com/dedmen){:target="_blank"} joined the project and Intercept has made a lot of progress since then.
```
Comment (Neviothr) [inline docs/_posts/2017-07-11-ace3-version3100.md]: `June` should be capitalized.

File: docs/_posts/2017-07-11-ace3-version3100.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+It has now reached a milestone where **100% of Arma 3 script commands** are available to be used and a lot of the core code was redesigned and refactored which also removed the latest hurdle in getting Intercept **whitelisted by BattlEye**.
+
+Besides just improving on existing code also new features have been added. The most notable being the ability to register ones own SQF commands so one can easily move small parts of ones SQF code directly to Intercept:,
```
Comment (Neviothr) [inline docs/_posts/2017-07-11-ace3-version3100.md]: `Ones` -> `One's`.

File: docs/_posts/2017-07-11-ace3-version3100.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+* [https://github.com/intercept/intercept/issues/10](https://github.com/intercept/intercept/issues/10){:target="_blank"}
+* [https://github.com/intercept/intercept/wiki/Registered-Functions](https://github.com/intercept/intercept/wiki/Registered-Functions){:target="_blank"}
+
+64bit support and Linux server compatability have been added to Intercept which was also important because now Intercept can use its full potential even on a server.
```
Comment (Neviothr) [inline docs/_posts/2017-07-11-ace3-version3100.md]: Should be `compatibility`.

**Comment by bux** [comment]
thx @Neviothr

**Pull request merged**